frontend/javascripts/admin/task/task_create_form_view.tsx

```diff
@@ -377,7 +376,10 @@ function TaskCreateFormView({ taskId, history }: Props) {
 
     if (taskId != null) {
       // either update an existing task
-      const newTask = { ..._.omit(formValues, "nmlFiles", "baseAnnotation"), boundingBox };
+      const newTask = {
+        ..._.omit(formValues, "nmlFiles", "baseAnnotation"),
+        boundingBox,
+      };
       const confirmedTask = await updateTask(taskId, newTask);
```

Comment:
Here the frontend sends too much information to the backend. Afaik the value `newTask` contains more fields than what the backend expects. But this state has been working for quite some time. @fm3 @daniel-wer Do you think I should change that and omit all unexpected fields before sending the request or keep it as it has been in the past?

Reply:
The backend generally ignores superfluous keys. I don’t have a strong opinion if the frontend should be cleaned up to send only the expected ones. Maybe Daniel has a preference :)

Reply:
No strong opinion either. You can leave it as-is imo.